src/massive/munit/TestRunner.hx

```diff
@@ -100,6 +100,8 @@ class TestRunner implements IAsyncDelegateObserver
     public var completionHandler:Bool -> Void;
 
     public var clientCount(get_clientCount, null):Int;
+    public var testSuites:Array<TestSuite>;
```

Comment:
I exposed it for my change in TestRunnerTest.hx, I could instead add a new method for the check but this seemed simplest.